surefire-its/src/test/resources/additional-classpath/src/test/java/additionalClasspath/BasicTest.java

```diff
@@ -12,4 +18,31 @@ public void testExtraResource()
         assertNotNull( BasicTest.class.getResourceAsStream( "/test2.txt" ) );
     }
 
+    public void testExtraResourceOrder()
+    {
+        Enumeration<URL> resources = BasicTest.class.getClassLoader().getResources("/order-test.txt");
+        assertTrue( resources.hasMoreElements() );
+        URL url = resources.nextElement();
+        InputStream is = url.openStream();
+        assertNotNull( is );
+        try ( InputStream i = is ) {
```

Comment:
There is no need for nested try with resources. One can open multiple objects in the same try: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
Sure; I always just find this way to be more readable. Is it important to change it?

Reply:
I went ahead and changed it.